Default Agent and Cluster-Agent to 7.48.0 version

adel121 opened the pull request:
#### What this PR does / why we need it:

Default Agent and Cluster-Agent to 7.48.0 version.


#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Chart Version bumped
- [ ] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [ ] `CHANGELOG.md` has been updated
- [ ] Variables are documented in the `README.md`
- [ ] For Datadog Operator chart or value changes update the test baselines (run: `make update-test-baselines`)

davidor reviewed (changes requested):
The version update looks good but you also introduced many formatting changes and some of them make the linter check fail.

adel121 commented:
> The version update looks good but you also introduced many formatting changes and some of them make the linter check fail.

The local formatter automatically applied some linting that caused the CI to fail. 
I just removed the linting that had been added. Should be good now.

Thank you
